omero/developers/json-api.rst

```diff
@@ -432,7 +432,7 @@ These query parameters are also supported (see above):
           "url:project": "http://server.openmicroscopy.org/api/v0/m/projects/11601/",
           "url:datasets": "http://server.openmicroscopy.org/api/v0/m/projects/11601/datasets/",
           "@id": 11601,
-          "@type": "http://www.openmicroscopy.org/Schemas/OME/2016-06#Project",
```

Comment:
Just realized this is not the case in `omero-marshal` although we should probably consider using HTTPS in  https://github.com/openmicroscopy/omero-marshal/blob/master/omero_marshal/__init__.py#L63 /cc @chris-allan

Reply:
I have on my todo list i.e. update omero-marshal

Reply:
We could certainly do this going forward but this breaks all the backwards compatibility. I would not change these.

Reply:
I will roll the change back in the example